website/content/docs/discovery/dns.mdx

```diff
@@ -398,7 +437,21 @@ of a service imported from that peer.
 
 The virtual IP is also added to the service's [Tagged Addresses](/docs/discovery/services#tagged-addresses)
 under the `consul-virtual` tag.
+ 
+### Service Virtual IP Lookups for Consul Enterprise <EnterpriseAlert inline />
```

Comment:
I think this should be demoted one level (since it's inside of the "Service Virtual IP Lookups" section at level `###`

```suggestion
#### Service Virtual IP Lookups for Consul Enterprise <EnterpriseAlert inline />
```